ledger-tool/src/main.rs

```diff
@@ -1060,6 +1061,31 @@ fn main() {
                              information that went into computing the completed bank's bank hash. \
                              The file will be written within <LEDGER_DIR>/bank_hash_details/",
                         ),
+                )
+                .arg(
+                    Arg::with_name("record_slots")
+                        .long("record-slots")
+                        .default_value("slots.json")
+                        // .default_value() does not work with .conflicts_with() in clap 2.33
+                        // .conflicts_with("verify_slots")
+                        // https://github.com/clap-rs/clap/issues/1605#issuecomment-722326915
```

Comment:
As you mentioned, would be nice if we were on recent CLAP so this bug didn't hinder us.

Instead, can we do the check ourselves instead. Ie, something like:
```rust
if arg_matches.is_present("record_slots") && arg_matches.is_present("verify_slots") {
    eprintln!("Some text, maybe mirror what the CLAP text would be if there weren't that bug");
}
```
What do you think ? Also, if we do this, we should do it fairly early in the "verify" match case, certainly before calling `load_and_process_ledger_or_exit!()`

Reply:
Yes, good point. I've added it.